client/src/Pages/Profile/UserInfo/index.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import {
+  Typography,
+  Avatar,
+  Grid,
+  Paper,
+  CircularProgress,
+} from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+
+import Button from '../../../Components/Button';
+
+import useStyles from './style';
+
+function UserInfo({ getUserName }) {
+  const classes = useStyles();
+  const [user, setUser] = useState({});
+  const [loading, setLoading] = useState('');
```

Comment:
This looks like it should be a boolean - it's useful to stick to the same types for the defaults eg. `useState(false)` otherwise you can get some tricky bugs creeping in